hadoop-ozone/dist/src/main/compose/ozonesecure/test.sh

```diff
@@ -43,8 +43,11 @@ for scheme in ofs o3fs; do
   done
 done
 
+exclude=""
 for bucket in encrypted link generated; do
-  execute_robot_test s3g -v BUCKET:${bucket} -N s3-${bucket} s3
+  execute_robot_test s3g -v BUCKET:${bucket} -N s3-${bucket} ${exclude} s3
+  # some tests are independent of the bucket type, only need to be run once
+  exclude="--exclude no-bucket-type"
```

Comment:
This `--exclude` - is this just a parameter that the `robot` command understands? We pass this through and it then excludes any tests with the tag no-bucket-type?

Reply:
Exactly.  I learned this from @smengcl in #3174.